pkg/sql/catalog/descs/descriptor.go

```diff
@@ -186,7 +186,7 @@ func (tc *Collection) getDescriptorsByID(
 		// TODO(ajwerner): Sort out the hydration mess; define clearly what is
 		// hydrated where and test the API boundary accordingly.
 		if table, isTable := desc.(catalog.TableDescriptor); isTable {
-			desc, err = tc.hydrateTypesInTableDesc(ctx, txn, table)
+			desc, err = tc.hydrateTypesInTableDescWitOptions(ctx, txn, table, flags.IncludeOffline, flags.AvoidLeased)
```

Comment:
I fixed this typo on master, so this needs a rebase, otherwise it'll skew and to adopt the fix, otherwise it'll skew